### PR TITLE
[BYORTL][Verilator] update ops and add MobileNet

### PR DESCRIPTION
**Pull request by vegaluisjose**
The following PR adds support to test end-to-end(sw/hw) MobileNet using the Verilator backend. This is build upon this [commit](https://github.com/apache/tvm-vta/commit/dfe9f572a43d41e0c1ecdf036cea97042a0febfe).

@tmoreau89

### CODE REVIEW

**Comment by tqchen** [comment]
Thanks @vegaluisjose . just want to make sure that the mobile net simulation won't cost a long time to finish. otherwise it would be still good to have unit test models that can finish in a reasonable amount of time. We can move mobile net tests to nightly category later once we support nightlies

**Comment by tmoreau89** [comment]
Thank you @vegaluisjose the changes LGTM. To alleviate @tqchen 's concerns can you comment on how long the simulations take on MobileNet? We're only simulating bias_add on a small RTL design so I presume the runtime won't be as long as some of the other TSIM tests on full VTA designs offloading entire conv2ds but let's make sure we keep tabs on runtime. Thanks!

**Comment by vegaluisjose** [comment]
It takes about the same time as running the pure software version.

**Comment by tmoreau89** [comment]
I tested the two tests on my local machine, `test_mobilenet.py` and `test_verilator_ops.py`; total runtime is 22.06s. @tqchen I'm going to approve the PR, but let us know if you think that is too long. IMO that is a reasonable runtime.

File: tests/python/contrib/test_verilator/test_mobilenet.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    return labels
+
+
+def check_result(res):
+    """Check prediction against labels."""
+    labels = get_labels()
+    predictions = np.squeeze(res)
+    prediction = np.argmax(predictions)
+    # 387 is the elephant
+    tvm.testing.assert_allclose(prediction, 387, rtol=1e-5, atol=1e-5)
```
Comment (tmoreau89) [inline tests/python/contrib/test_verilator/test_mobilenet.py:187]: In this case can't we just assert that prediction is equal to 387?

**Review by tmoreau89 — APPROVED** [review]
Thanks @vegaluisjose ; I left a nit that would be best to address if you have the bandwidth.

File: tests/python/contrib/test_verilator/test_mobilenet.py (hunk 1 above)
Comment (vegaluisjose) [reply tests/python/contrib/test_verilator/test_mobilenet.py]: Fixed

**Comment by tmoreau89** [comment]
Weird, it looks like we got a CI error, but probably unrelated to the changes. Could you re-trigger @vegaluisjose ?

**Comment by tmoreau89** [comment]
Thank you @vegaluisjose the PR has been merged.

**Pull request merged**